### PR TITLE
CI: Set up Travis

### PR DESCRIPTION
**Pull request by Hywan**
Address https://github.com/hoaproject/Central/issues/36.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+before_script:
+  - export PATH="$PATH:$HOME/.composer/vendor/bin"
+  - if [[ ! $ENABLE_XDEBUG ]]; then
+      phpenv config-rm xdebug.ini || echo "ext-xdebug is not available, cannot remove it.";
+    fi
+
+script:
+  - composer install
+  - vendor/bin/hoa test:run
```
Comment (vonglasow) [inline .travis.yml]: shouldn't we add `travis_wait` for this step while using xdebug due to latency?

Comment (Hywan) [reply .travis.yml]: Already tried for `hoa/json` but it takes too long…

**Comment by mfn** [comment]
Hi,
what is needed to move forward with this?

From looking at the latest travis builds:
- 5.5 fails => but is this still relevant? [It has reached EOL 1 1/2 years ago](http://php.net/eol.php)
- 7.1 with xdebug fails => seems like a timeout, it's known (and also mentioned above) that it slows PHP down too much at times

**Comment by Hywan** [comment]
5.5 has been dropped, and for 7.1, yes, xdebug makes it fail unfortunately because of a timeout. Maybe we will drop `ENABLE_DEVTOOLS` for this test matrix.

**Comment by mfn** [comment]
Thank you 😄

**Pull request merged**